Examples/Framework/include/ActsExamples/EventData/SimSourceLink.hpp

```diff
@@ -44,7 +44,7 @@ class SimSourceLink {
   constexpr const Acts::Surface& referenceSurface() const { return *m_surface; }
   constexpr const ActsFatras::Hit& truthHit() const { return *m_truthHit; }
 
-  Acts::FittableMeasurement<SimSourceLink> operator*() const {
+  Acts::FittableMeasurement<SimSourceLink> makeMeasurement() const {
```

Comment:
Cosmetic change

Reply:
This PR removes the requirement for source links to be dereferenceable. That means there is not need to have this functionality implemented as a special operator overload anymore.

Reply:
But it replaces it with the implicit requirement for there to be `makeMeasurement` method, no?

Reply:
No, because the calibrator is not generic, but is usually specific for a given source link type.